dcc-portal-ui/app/scripts/software/js/software.js

```diff
@@ -42,10 +42,18 @@
       Page.setPage('entity');
       Page.setTitle('Software Downloads');
 
-      $scope.artifactId = 'icgc-storage-client';
-      jQuery.get('api/v1/ui/software/' + $scope.artifactId + '/versions', function(versions) {
+      $scope.icgcStorageClient = {
+    		  artifactId: 'icgc-storage-client'
+      };
+      $scope.icgcGet = {
+    		  artifactId:'icgc-get'
+      };
+      jQuery.get('api/v1/ui/software/' + $scope.icgcStorageClient.artifactId + '/versions', function(versions) {
```

Comment:
I realize this wasn't originally written by you, but I think we should use either `Restangular` here or `$resource`. Please speak with @cheapsteak tomorrow.